CONTRIBUTING.md

```diff
@@ -30,7 +30,7 @@ brew cask install docker
 
 On Ubuntu:
 ```bash
-apt-get install cmake pkg-config libicu-dev docker-ce
+apt-get install cmake pkg-config libicu-dev docker.io ruby ruby-dev libz-dev build-essential libssl-dev
```

Comment:
Looks like libz-dev has been replaced:

```console
# apt-get install cmake pkg-config libicu-dev docker.io ruby ruby-dev libz-dev build-essential libssl-dev
Reading package lists... Done
Building dependency tree
Reading state information... Done
Note, selecting 'zlib1g-dev' instead of 'libz-dev'  <----- HERE
The following additional packages will be installed:
[...]
```

```suggestion
apt-get install cmake pkg-config libicu-dev docker.io ruby ruby-dev zlib1g-dev build-essential libssl-dev
```